src/app/archive/dune

```diff
@@ -6,24 +6,4 @@
  (modes native)
  (libraries archive_cli async async_unix core_kernel base mina_version bounded_types)
  (instrumentation (backend bisect_ppx))
- (preprocess (pps ppx_version)))
```

Comment:
I suggest to extract this change to a separate PR.

Library `archive_cli` on which `archive` depends contains an implicit dependency on `mina_signature_kind`, hence it's potentially risky to not specialize the signature kind at the executable level.

I'd prefer `archive_cli` to get rid of `mina_signature_kind` dependency and then this change will start to look safe.

P.S. to check that `mina_signature_kind` is a dependency I used the following commands from `georgeee/granular-nix` branch:

```bash
nix build mina#info.deps --out-link deps.json
<deps.json jq '.units.archive.lib."archive.cli".pkgs | contains(["mina_signature_kind"])'
```